### PR TITLE
Change all references to minio.io to min.io

### PR DESCRIPTION
**Pull request by harshavardhana**

### CODE REVIEW

File: gluegun.gemspec [hunk 1]
```diff
@@ -32,7 +32,7 @@ Gem::Specification.new do |spec|
 
   spec.post_install_message = %q{
     Generate sites using:  `./gluegun generate site.yml`
-    For more details see https://gluegun.minio.io
+    For more details see https://gluegun.min.io
```
Comment (kaankabalak) [inline gluegun.gemspec:35]: This link doesn't seem to be leading anywhere, and the older link just redirects to our current docs, we might think of either rewording this message or redirect users to our repo on minio/gluegun (cc: @deekoder)

I think we would also need to make a similar change on minio/gluegun as this is essentially the specification of our gem.

File: nginx/default [hunk 2]
```diff
@@ -15,7 +15,7 @@ server {
 
     # Add index.php to the list if you are using PHP
     index index.html index.htm index.nginx-debian.html;
-    server_name gluegun.minio.io; # managed by Certbot
+    server_name gluegun.min.io; # managed by Certbot
```
Comment (kaankabalak) [inline nginx/default:18]: Similar to my previous comment, gluegun.minio.io leads to our Docs site, but gluegun.min.io doesn't lead anywhere. If this is the setting to configure this redirect, this comment can be ignored.

File: gluegun.gemspec (hunk 1 above)
Comment (harshavardhana) [reply gluegun.gemspec:35]: gluegun.min.io is added to our DNS @kaankabalak

File: nginx/default (hunk 2 above)
Comment (harshavardhana) [reply nginx/default:18]: gluegun.min.io is added to our DNS @kaankabalak

**Review by kaankabalak — APPROVED** [review]

**Review by deekoder — APPROVED** [review]

**Pull request merged**